src/Cli/dotnet/Commands/Test/TestingPlatformCommand.cs

```diff
@@ -107,7 +112,7 @@ private void SetupCancelKeyPressHandler()
         Console.CancelKeyPress += (s, e) =>
         {
             _output?.StartCancelling();
-            CompleteRun();
+            CompleteRun(exitCode: ExitCode.Aborted);
```

Comment:
Are we sure this will be the real exit code of the `dotnet.exe` process?

Reply:
not 100% sure, I can also just pass in null.

Reply:
Okay let's pass null then. I want us to be sure that what we present is the reality of the exit code of `dotnet test`